CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.3.0]
+## [0.4.0] - 2020-04-01
+- Changed: update dropwizard-core version to 4.1.5
+- Changed: update to NewRelic telemetry SDK 0.4.0
```

Comment:
We should also include the additional of the contributed `NewRelicReporterFactory`

Reply:
Ok, I added. Not sure on the convention...

Reply:
looks great!